Models/Tip.cs

```diff
@@ -6,9 +6,8 @@ namespace SmartSaver.Models
 {
     public class Tip
     {
-
-        private string tipName;
-        private string tipLink;
+        public string tipName { get; set; }
+        public string tipLink { get; set; }
```

Comment:
Only a small naming convention drawback : ) Should be named in CamelCase (TipName, TipLink)